### PR TITLE
JENKINS-64373 Add additional help locator fallback

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-64373](https://issues.jenkins-ci.org/browse/JENKINS-64373).

Fixes:
image

When clicking the first help icon in a trait from github-branch-source

image

After this change it correctly expands:

image

The next tr doesn't have either of the additional locator's that are required for the current code to work:

image

Going back to before tables to divs it was only the one class required.

Not sure if @jsoref or @fqueiruga can remember why it was changed but having this as the last fallback doesn't seem to cause any issues, and fixes at least this bug.

Another alternative would be tracking down what is creating that help icon and adding the class

### Proposed changelog entries

* Entry 1: JENKINS-64373, Help text now expands in behaviours for GitHub organization folders
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/webapp/scripts/hudson-behavior.js
Comment (jsoref) [inline war/src/main/webapp/scripts/hudson-behavior.js:757]: Should this be indented one more space?
```suggestion
             findFollowingTR(this, "help-area");
```

**Review by jsoref — APPROVED** [review]

**Review by fqueiruga — APPROVED** [review]
Looks good, although it would be good to fish for the original change and see if that case still works (it should I know)

**Comment by timja** [comment]
> Looks good, although it would be good to fish for the original change and see if that case still works (it should I know)

Given this is adding a fallback as last priority I can’t see it causing an issue, I’ve clicked around in a few places and all seems to work

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by res0nance — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**
